### ISSUE TITLE
Add Check For Updates Button , Auto Check For Updates Before Starting Software

### ISSUE THREAD

**Issue by hudson1740** [Brett-Tech-Networking/Moradi-Notepad#22]
Add available option to check for software updates with a button and or before the software starts (splash screen with say 15-20 sec load bar would be idea for background process of checking for updates), either direct releases or github commits

**Comment by MattMoradi** [comment]
This integration is currently on hold until Issue #40 is resolved. :/

**Comment by hudson1740** [comment]
I think this can be closed as it is already implemented with the exception of a few tweaks

**Comment by MattMoradi** [comment]
Not quite yet actually. I'm still working on it for notepad.
